worlds/stardew_valley/test/TestMods.py

```diff
@@ -0,0 +1,66 @@
+from . import setup_solo_multiworld
+from BaseClasses import ItemClassification, MultiWorld
```

Comment:
Things are failing here
![image](https://github.com/agilbert1412/Archipelago/assets/16685738/d55ac209-b3ef-41cb-8369-c2b38371963f)

Reply:
Was due to improperly typing the import.